hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,82 +10,95 @@ This module conforms to the [Comportable guideline for peripheral functionality.
 
 ## Features
 
-- This revision supports a LFSR (linear feedback shift register) digital source, and provides an interface to an external rng (random noise generator) source.
-The LFSR is used as a digital, pseudo-random type of entropy source, while the rng external source is a true entropy source.
+- This revision provides an interface to an external RNG (random noise generator) source, and a LFSR (linear feedback shift register) digital source. 
+The LFSR is used as a digital, pseudo-random type of entropy source, while the RNG external source is a digital noise source.
```

Comment:
can i ask why you changed from `true` to `digital`? I feel like digital almost implies its yet another prng :)

Reply:
unfortunately there are many similar terms in this security world. I used the term straight out of the NIST 90B diagram, which exactly matches our configuration. @martin-lueker has spent a fair amount of time bridging NIST and CC terms, so no small task. If you feel strong that this is not accurate i can change it, but trying to get the best terms here. Maybe @martin-lueker has a preferred term.

Reply:
ah no worries. If there's clear documentation for why the other term is used, i'm sure there are good reasons.

Reply:
This sounds ok to me, the "noise" is the critical part.